### PR TITLE
Run acl test twice - with and without flow logging 

### PR DESCRIPTION
**Pull request by milan-zededa**
Soon EVE will allow to disable flow logging (and it will become the default behavior),
which reduces the amount of data published to the controller and simplifies iptables
rules installed by EVE.
See: https://github.com/lf-edge/eve-api/pull/62

However, the ACL implementation is quite different between enabled and
disabled flow logging. Therefore, it makes sense to run the ACL test twice
to test both cases.

### CODE REVIEW

File: tests/workflow/networking.tests.txt [hunk 1]
```diff
@@ -32,25 +32,27 @@ eden.escript.test -test.run TestEdenScripts/eden_onboard
 eden.escript.test -test.run TestEdenScripts/template_check
 {{end}}
 
-/bin/echo Eden basic network test (5/{{$tests}})
-eden.escript.test -testdata ../network/testdata/ -test.run TestEdenScripts/network_test
+#/bin/echo Eden basic network test (5/{{$tests}})
+#eden.escript.test -testdata ../network/testdata/ -test.run TestEdenScripts/network_test
```
Comment (giggsoff) [inline tests/workflow/networking.tests.txt:36]: They are not required anymore? Or it is some lines come from your debugging process?

Comment (milan-zededa) [reply tests/workflow/networking.tests.txt]: Good catch! I left it there accidentally.
It is fixed now.

**Review by uncleDecart — APPROVED** [review]
LGTM, do we need to bump EVE version as well, since we're updating API? 

Edit: Okay, we can merge once EVE is released and then we will have to release new Eden

**Comment by milan-zededa** [comment]
> LGTM, do we need to bump EVE version as well, since we're updating API?
> 
> Edit: Okay, we can merge once EVE is released and then we will have to release new Eden

No need, test passes also for older EVE.
Actually, I would prefer to first merge this, release eden and bring this to EVE. Otherwise, my next EVE PR will fail networking tests.

**Pull request merged**